src/py/flwr/common/config.py

```diff
@@ -71,3 +71,44 @@ def get_project_config(project_dir: Union[str, Path]) -> Dict[str, Any]:
         )
 
     return config
+
+
+def flatten_dict(
+    raw_dict: Dict[str, Any], parent_key: str = "", sep: str = "."
+) -> Dict[str, str]:
+    """Flatten dict by joining nested keys with a given separator."""
+    items: List[Tuple[str, str]] = []
+    for k, v in raw_dict.items():
+        new_key = f"{parent_key}{sep}{k}" if parent_key else k
+        if isinstance(v, dict):
+            items.extend(flatten_dict(v, parent_key=new_key, sep=sep).items())
+        elif isinstance(v, str):
+            items.append((new_key, v))
+        else:
+            raise ValueError(
+                f"The value for {k} needs to be a string.",
+            )
+    return dict(items)
+
+
+def parse_config_args(
+    config_overrides: Optional[str],
+) -> Dict[str, str]:
+    """Parse comma separated list of key-value pairs separated by '='."""
+    overrides: Dict[str, str] = {}
+
+    if config_overrides is not None:
```

Comment:
```suggestion
    if config_overrides is None:
        return overrides
```
...and then unindent the rest of the function